docs/src/user-guide/guides-retention.md

```
@@ -0,0 +1,167 @@
+  if (T - search_result.T > TTL) then EXPIRED
+  ```
+
+---
+
+## Configuration
+CLP allows users to configure different **retention_periods** for different types of data. 
+Additionally, a custom **sweep_interval** can be set for each type of data, which determines how
+frequently its garbage collection job is executed. These settings can be configured in 
+`etc/clp-config.yml`. 
+
+### Configure retention period
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)